### DISCUSSION TITLE
Angular text with an attribute disabled

### DISCUSSION THREAD

**Discussion by mboiman** [getgauge/taiko#2297] [General]
Hi, i have a problem with an angular button, wich is disabled. but with attribute and design. not over html

It has this code
`<ds-text _ngcontent-ygm-c48="" class="screensize-small style-inherit hydrated" app-disabled="" disabled="true"> text</ds-text>`

how can I check with taiko for the status of disabled?

I can find it and highlight it, but no idea how to see the status disabled="true"

**Comment by zabil** [comment]
If you just want to check the attribute you could run a script in the DOM on the element and return a true or false.
Please refer https://docs.taiko.dev/running_scripts_inside_the_browser/#read-elements-and-it's-attributes

**Comment by saikrishna321** [comment]
@mboiman you can try 

Option1:

```$('.screensize-small').isDisabled()```

Options2:

```$('.screensize-small').getAttribute('disabled')```

**Comment by mboiman** [comment]
thanks I am trying, if I have a solution I will post it
